### ISSUE TITLE
Define participation-adjusted inflation curve

### ISSUE THREAD

**Issue by Zerim** [graphprotocol/research#70]
The protocol will implement a participation reward, paid via inflation, which varies according to how many tokens are staked productively in the network.

We must define the curve which relates the `participationReward` to `targetParticipation` and `actualParticipation`, and any additional parameters for that curve which may be set via governance.

Requirements:
-  if `actualParticipation` === `targetParticipation`, then `participationRewardRate = 0`.
- if `actualParticipation === 0` then `participationRewardRate = 0`. (There is noone to receive the award).

**Comment by jtremback** [comment]
Is this to say that the total participation reward is just split up among all the participating parties?

**Comment by jtremback** [comment]
I'm having a tough time relating this to stuff in https://github.com/graphprotocol/research/tree/master/specs/graph-protocol-v1/mechanism-design .

In this issue when you refer to `participationReward` do you actually mean `participationRewardRate`?

**Comment by Zerim** [comment]
@jtremback Correct, reworded `participationReward` --> `participationRewardRate` above.
